lesson26/src/index.html

```diff
@@ -9,6 +9,9 @@
     <title>コンテンツページ</title>
   </head>
   <body class="bg-white font-indexBody">
+        <script>
+          window.location.href = "./login.html";
+        </script>
```

Comment:
This is a script for screen transitions. After this, we will implement it to determine if there is a token or not.